src/sources/aws_s3/mod.rs

```diff
@@ -690,14 +778,26 @@ mod integration_tests {
             let events = collect_n(rx, expected_lines.len()).await;
 
             assert_eq!(expected_lines.len(), events.len());
-            for (i, event) in events.iter().enumerate() {
-                let message = expected_lines[i].as_str();
-
-                let log = event.as_log();
-                assert_eq!(log["message"], message.into());
-                assert_eq!(log["bucket"], bucket.clone().into());
-                assert_eq!(log["object"], key.clone().into());
-                assert_eq!(log["region"], "us-east-1".into());
+            if !log_namespace {
```

Comment:
It would be nice if this used the new functions from https://github.com/vectordotdev/vector/pull/15054 so you don't need the if statement / duplication of assertions here